blocks/image-compare/src/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Image Compare Block
+ * A block that allows inserting side-by-side image comparison.
+ */
+
+
+// WordPress dependencies
+import { registerBlockType } from '@wordpress/blocks';
+import { RichText } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+
+export function registerBlock() {
+
+	registerBlockType( 'jetpack/image-compare-block', {
+
+		title: 'Image Compare',
+
+		icon: 'image-flip-horizontal',
+
+		category: 'layout',
+
+		attributes: {
+			imageBefore: {
+				type: 'string',
+				source: 'attribute',
+				attribute: 'src',
+				selector: '.imgBefore',
+			},
+			imageAfter: {
+				type: 'string',
+				source: 'attribute',
+				attribute: 'src',
+				selector: '.imgAfter',
+			},
+			caption: {
+				type: 'string',
+				source: 'html',
+				selector: 'figcaption',
+			},
+			orientation: {
+				type: 'string',
+			}
+		},
+
+		example: {
+			attributes: {
+				imageBefore: 'https://placedog.net/240/120',
```

Comment:
We need to ensure that we are allowed not only to use these images (license), but that their website allows hotlinking.

Wikimedia does. We could use these two photos of Lynda Carter as Wonder Woman:

<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6b/Lynda_Carter_Wonder_Woman_black_and_white.jpg/820px-Lynda_Carter_Wonder_Woman_black_and_white.jpg">
https://commons.wikimedia.org/wiki/File:Lynda_Carter_Wonder_Woman_black_and_white.jpg

<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d8/Lynda_Carter_Wonder_Woman.JPG/820px-Lynda_Carter_Wonder_Woman.JPG">
https://commons.wikimedia.org/wiki/File:Lynda_Carter_Wonder_Woman.JPG

Wikimedia explicitly _allows_ hotlinking, and those two pictures are public domain, so we're as safe as can be.

Reply:
Switched.